xcodeproj/internal/xcodeproj.template.bazelrc

```diff
@@ -15,6 +15,9 @@ build:rules_xcodeproj --experimental_action_cache_store_output_metadata
 # Xcode builds shouldn't mess with the workspace's symlinks
 build:rules_xcodeproj --experimental_convenience_symlinks=ignore
 
+# Ensure all our intermediate files are downloaded with BwtB
+build:rules_xcodeproj --experimental_remote_download_regex=.*\\.indexstore/.*|.*\\.swiftdoc$|.*\\.swiftmodule$|.*\\.swiftsourceinfo$
```

Comment:
Is there a way to filter this for just the target platform? I think this would also force downloading of Swift tools' swiftmodules etc. (though I don't think it would make any significant difference)

Reply:
I'll look into that :+1: